tests/solvers/test_radau5.py

```diff
@@ -732,7 +734,9 @@ def test_atol(self):
         
         nose.tools.assert_equal(steps3, steps2)
         
-        nose.tools.assert_raises(Radau_Exception, self.sim._set_atol, [1e-6,1e-6,1e-6])
+        err_msg = f"atol must be of length one or same as the dimension of the problem."
```

Comment:
You are using an f-string here but not formatting the string, I am just adding the comment here in case you missed something.